src/sam_bot_nav2_gz/requirements.txt

```diff
@@ -1,2 +1,2 @@
 artefacts_cli
-artefacts_toolkit>=0.3.1
+artefacts_toolkit>=0.3.3
```

Comment:
Actually this will probably break as it is `artefacts_toolkit_rosbag` that updated to 0.3.3 

The toolkit itself hasnt updated yet, just its module. The next version will be 0.4.0 (as rosbag has a new feature)

so this can be reverted to how you had it